cypress/integration/pages/mediaAssetPage/helpers.js

```diff
@@ -14,3 +16,29 @@ export const getBlockData = (blockType, body) => {
 
   return getBlockByType(blocks, blockType);
 };
+
+export const hasMedia = body => {
+  const mediaTypes = ['video', 'version', 'media', 'legacyMedia'];
+  return mediaTypes.some(type => body.metadata.blockTypes.includes(type));
+};
+
+const getServiceId = body => {
+  const mediaBlock = body.promo.media;
+  const versionId = mediaBlock.versions && mediaBlock.versions[0].versionId;
```

Comment:
Ramda has a function that might simplify this logic: https://ramdajs.com/docs/#paths

Something like:
```
const [vpid, externalId, id] = paths([
  ['versions', '0', 'versionId'],
  ['externalId'],
  ['id']
], mediaBlock)

return vpid || externalId || id;
```